staging/operator-lifecycle-manager/pkg/controller/registry/reconciler/grpc.go

```diff
@@ -146,6 +146,16 @@ func (c *GrpcRegistryReconciler) currentService(source grpcCatalogSourceDecorato
 	return service
 }
 
+func (c *GrpcRegistryReconciler) currentServiceAccount(source grpcCatalogSourceDecorator) *corev1.ServiceAccount {
+	serviceAccountName := source.ServiceAccount().GetName()
+	serviceAccount, err := c.Lister.CoreV1().ServiceAccountLister().ServiceAccounts(source.GetNamespace()).Get(serviceAccountName)
+	if err != nil {
+		logrus.WithField("service", serviceAccount).Debug("couldn't find service in cache")
```

Comment:
Do we need to pull in the #297 commit as well here?

Reply:
Disregard - wrong PR (:laughing:)

Reply:
Addressed in https://github.com/openshift/operator-framework-olm/pull/298